docs/root/configuration/http/http_filters/thrift_to_metadata_filter.rst

```diff
@@ -0,0 +1,49 @@
+.. _config_http_filters_thrift_to_metadata:
+
+Envoy Thrift-To-Metadata Filter
+===============================
+* This filter should be configured with the type URL ``type.googleapis.com/envoy.extensions.filters.http.thrift_to_metadata.v3.ThriftToMetadata``.
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.thrift_to_metadata.v3.ThriftToMetadata>`
+
+This filter is configured with rules that will be matched against Apache thrift compatible requests and responses.
+The filter will parse the thrift body, extract thrift metadata or payload, and add them to dynamic metadata
+based on the configuration of the rule.
```

Comment:
Again, make it clear that this if for thrift requests/responses within the context of HTTP.